operator/roles/kiali-deploy/tasks/main.yml

```diff
@@ -5,32 +5,32 @@
   set_fact:
     api_groups: "{{ lookup('k8s', cluster_info='api_groups') }}"
   when:
-  - is_openshift == False
-  - is_k8s == False
+    - is_openshift == False
```

Comment:
Revert all the changes made to indentation and quoting in this file. The only change that needs to be included in this PR is lines 120-140 (while of course maintaining the proper indentation and quoting standards)

Reply:
Reverted this, These changes were not intended the editor auto formatted this on save